### ISSUE TITLE
Can you give the setup instruction in plain english?

### ISSUE THREAD

**Issue by theodorsudarja** [Rod-Persky/investoscope-easy-update#2]
Hi Rod Persky,

Can you give the setup instruction in plain english? I don't even know where to start.
I read the instruction on the page. But I don't have any idea.

**Comment by Rod-Persky** [comment]
+1 will fix :)

**Comment by Rod-Persky** [comment]
@theodorsudarja I've now made Investoscope Easy Update an apple application that can simply be opened without any command line kung-fu 👍 maybe you can have another go?
